level/level.go

```diff
@@ -100,6 +129,36 @@ func allowed(allowed level) Option {
 	return func(l *logger) { l.allowed = allowed }
 }
 
+// Parse a string to it's corresponding level value. Valid strings are "debug",
+// "info", "warn", and "error". Strings are normalized via strings.TrimSpace and
+// strings.ToLower.
+func Parse(level string) (Value, error) {
+	switch strings.TrimSpace(strings.ToLower(level)) {
+	case debugValue.name:
+		return debugValue, nil
+
+	case infoValue.name:
+		return infoValue, nil
+
+	case warnValue.name:
+		return warnValue, nil
+
+	case errorValue.name:
+		return errorValue, nil
+
+	default:
```

Comment:
Stylistically I don't think the blank lines between cases are warranted when each is so simple. I suggest we remove them.